src/workflows/context/context.py

```diff
@@ -480,17 +489,9 @@ async def synthesize(
 
         return None
 
-    def send_event(self, message: Event, step: str | None = None) -> None:
-        """Dispatch an event to one or all workflow steps.
-
-        If `step` is omitted, the event is broadcast to all step queues and
-        non-matching steps will ignore it. When `step` is provided, the target
-        step must accept the event type or a
-        [WorkflowRuntimeError][workflows.errors.WorkflowRuntimeError] is raised.
-
-        Args:
-            message (Event): The event to enqueue.
-            step (str | None): Optional step name to target.
+    def _send_event(self, message: Event, step: str | None = None) -> None:
```

Comment:
Let's keep this around as `send_event` and make it a shallow wrapper of `send_events`.